### ISSUE TITLE
InfoBatch for image retrieval？

### ISSUE THREAD

**Issue by Tong-Jin01** [NUS-HPC-AI-Lab/InfoBatch#29]
I am trying to use InfoBatch for an image retrieval task. The loss I calculated is a tensor value (tensor(1.5947, device='cuda:0', grad_fn=<MeanBackward0>)), which has no ***.shape*** attribute, so when I call ***loss = train_dataset.update(loss)***, I get an error. How can I solve this problem?

![Image](https://github.com/user-attachments/assets/bda4ca00-7669-401a-97db-68032e132311)

![Image](https://github.com/user-attachments/assets/f235ba1d-62ec-4d35-acfa-f80f51d1c570)
